libs/dal/src/dal.service.ts

```diff
@@ -1,15 +1,11 @@
-import { Connection, ConnectionOptions } from 'mongoose';
+import { Connection, ConnectOptions } from 'mongoose';
 import * as mongoose from 'mongoose';
 
 export class DalService {
   connection: Connection;
 
-  async connect(url: string, config: ConnectionOptions = {}) {
-    const instance = await mongoose.connect(url, {
-      ...config,
-      autoReconnect: true,
```

Comment:
A small question, why the autoReconnect here was removed? When a cluster does a failover, it won't be able to connect and will crash the process.

Reply:
The "autoReconnect" doesn't exist anymore, here is current reconnect logic:
https://mongoosejs.com/docs/connections.html#error-handling

Reply:
Got ya, thanks!